lib/rx/client.rb

```diff
@@ -86,6 +86,10 @@ def connection
       end
     end
 
+    def base_path
```

Comment:
https://github.com/department-of-veterans-affairs/devops/issues/588#issuecomment-253232495 contains a path that differs from the base path configured for RX. This change configures the core `RX::Client` to use the `/patient/v1/` endpoint, while using the same `RX::Configuration` for both `RX::Client` and `RX::HealthCheckClient`.

I wonder if it would be better to use `RX::HealthCheckClient < RX::Client` and `RX::PatientClient < RX::Client`, but I think that would introduce too large a refactor for the current purposes.